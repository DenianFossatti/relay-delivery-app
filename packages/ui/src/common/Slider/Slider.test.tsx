import { mockTestID, render, wrapper } from '../../tests';

import { Slider, SliderProps } from '.';

describe(`Slider`, () => {
  const props: SliderProps = {
    testID: 'Slider',
  };

  const { findByTestId, toJSON } = render(<Slider {...props} />, { wrapper });

  const el_container = mockTestID('Slider', props.testID!);

  it(`render component #${el_container}`, async () => {
    const sut = await findByTestId(el_container);
    expect(sut).toBeTruthy();
  });

  it('should render snapshot', () => {
    expect(toJSON()).toMatchSnapshot();
  });
});
