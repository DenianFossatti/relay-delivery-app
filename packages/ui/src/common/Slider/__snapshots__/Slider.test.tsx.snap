// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/Slider should render snapshot 1`] = `null`;
